chimg/chroot.py

```diff
@@ -87,6 +88,30 @@ def _cmd_run(self, cmd: str) -> Tuple[str, str]:
             if fname:
                 os.remove(fname)
 
+    def _snaps_base_install(self, snap_infos: Dict[str, SnapInfo]) -> None:
+        """
+        Install the required core/coreXX snaps for the given snaps
+        """
+        # install the required base snaps
+        required_cores = set()
+        for snap, si in snap_infos.items():
+            if snap == "snapd":
+                # snapd is self-contained, ignore base
+                continue
+            if re.match(r"^(core|core\d{0,2})$", snap):
```

Comment:
You can drop `core|` because `core\d{0,2}` matches `core`, `core2` and `core22`. Alternatively for more strictness `core(?:\d\d)?`.